packages/edit-post/src/components/header/header-toolbar/style.scss

```diff
@@ -15,6 +15,16 @@
 
 	.edit-post-header-toolbar__left > .edit-post-header-toolbar__inserter-toggle {
 		display: inline-flex;
+
+		svg {
+			transition: transform cubic-bezier(0.165, 0.84, 0.44, 1) 0.2s;
+		}
+
+		&.is-pressed {
+			svg {
+				transform: rotate(45deg);
```

Comment:
If we go and keep this, it might be nice to update the Figma "small x" icon to be more or less identical to the plus as rotated.

Reply:
Agreed, let's assess in a follow up.

Reply:
(It might already be!)

Reply:
I checked, it's close but no cigar 😅

<img width="748" alt="Screenshot 2021-03-17 at 15 01 19" src="https://user-images.githubusercontent.com/846565/111488989-a8cbae00-8731-11eb-989e-c5941fb43019.png">

Reply:
Hah, I bet that when the pixels antialias together the difference is barely perceptible.

Of course that won't be good enough for you or me, because _we'll know_! But I'm happy to push the vectors once we get to it 😄